### ISSUE TITLE
[Bug]: GraphQL returns error when ElementCounter has no objects to count

### ISSUE THREAD

**Issue by afe038** [pimcore/data-hub#487]
### Expected behavior

Executing the following query:

```
{
  getSuppliersListing(
    published: true
    sortBy: "SupplierName"
    sortOrder: "ASC"
    ids: "3265"
  ) {
    edges {
      node {
        key
        id
        countProducts
        countCategories
      }
    }
  }
}
```

Should return:

```
{
  "data": {
    "getSuppliersListing": {
      "edges": [
        {
          "node": {
            "key": "Some Supplier",
            "id": "3265",
            "countProducts": null,
            "countCategories": null
          }
        }
      ]
    }
  }
}
```

### Actual behavior

I have Supplier and Product objects related, I have created an ElementCounter that will count for Products assigned to a Supplier. I have the following query:

```
{
  getSuppliersListing(
    published: true
    sortBy: "SupplierName"
    sortOrder: "ASC"
    ids: "3265"
  ) {
    edges {
      node {
        key
        id
        countProducts
        countCategories
      }
    }
  }
}
```

However, because some suppliers don't have any products assigned, the query returns:

```
{
  "errors": [
    {
      "message": "Warning: Attempt to read property \"value\" on null"
    }
  ]
}
```

I have solved this issue by adding a check in the following function on the file ./vendor/pimcore/data-hub/src/GraphQL/Query/Operator/ElementCounter.php:

`    

    public function getLabeledValue($element, ResolveInfo $resolveInfo = null)
    {
        $result = new \stdClass();
        $result->label = $this->label;

        // Pimcore 5/6 compatibility
        $children = method_exists($this, 'getChildren') ? $this->getChildren() : $this->getChilds();
        $count = 0;

        foreach ($children as $c) {
            $valueResolver = $this->getGraphQlService()->buildValueResolverFromAttributes($c);

            $childResult = $valueResolver->getLabeledValue($element, $resolveInfo);
            // Added check for null
            if($childResult == null){
                    return 0;
            }
            else{
                $childValues = $childResult->value;
            }
            // end of fix

            /* OLD CODE
             * $childValues = $childResult->value;
            */

            if ($this->getCountEmpty()) {
                if (is_array($childValues)) {
                    $count += count($childValues);
                } else {
                    $count++;
                }
            } else {
                if (is_array($childValues)) {
                    foreach ($childValues as $childValue) {
                        if ($childValue) {
                            $count++;
                        }
                    }
                } elseif ($childValues) {
                    $count++;
                }
            }
        }

        $result->value = $count;

        return $result;
    }

`

### Steps to reproduce

Add ElementCounter for a many-to-many relation that don't have objects assigned

**Comment by FredoVelcro** [comment]
Hello, 
Bug confirmed, patch should be instead :  
```php
if($childResult == null){
                $count = 0;
                break;
            }
            else{
                $childValues = $childResult->value;
            }
```

**Comment by mcop1** [comment]
@afe038 @FredoVelcro would one of you guys mind and contribute by opening a pull request, please? Would be really appreciated. 

Thank you for your efforts

**Comment by aryaantony92** [comment]
Fixed by #522
